src/plugins/intel_gpu/src/kernel_selector/kernels/gemm/gemm_kernel_tiled_opt.cpp

```diff
@@ -184,7 +224,7 @@ bool GemmKernelTiledOpt::Validate(const Params& params, const optional_params& o
     const auto& gmm_params = static_cast<const gemm_params&>(params);
     bool gemm_leftovers = gmm_params.inputs[0].X().v % 16 || gmm_params.inputs[0].Y().v % 16 ||
                           gmm_params.inputs[1].X().v % 16 || gmm_params.inputs[1].Y().v % 16;
-    if ((gmm_params.transpose_input0 || gmm_params.transpose_input1) && gemm_leftovers)
+    if ((gmm_params.transpose_input0 || gmm_params.transpose_input1) && (gemm_leftovers || gmm_params.has_dynamic_inputs()))
```

Comment:
Is there any fundamental restriction for the dynamic impl for transposed cases or just to leave it to be done in the future?

Reply:
Yes. If gmm_params has dynamic inputs, leftovers cannot be calculated. Added corresponding comment.